custom/conf/app.example.ini

```diff
@@ -550,34 +549,32 @@ ENABLE = true
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Main Logger
 ;;
-;; Either "console", "file", "conn", "smtp" or "database", default is "console"
+;; Either "console", "file" or "conn", default is "console"
 ;; Use comma to separate multiple modes, e.g. "console, file"
 MODE = console
 ;;
 ;; Either "Trace", "Debug", "Info", "Warn", "Error", "Critical" or "None", default is "Info"
 LEVEL = Info
 ;;
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; Router Logger
-;;
-;; Switch off the router log
-;DISABLE_ROUTER_LOG=false
+;; Print Stacktrace with logs (rarely helpful, do not set) Either "Trace", "Debug", "Info", "Warn", "Error", "Critical", default is "None"
+;STACKTRACE_LEVEL = None
 ;;
-;; Set the log "modes" for the router log (if file is set the log file will default to router.log)
-ROUTER = console
+;; Buffer length of the channel, keep it as it is if you don't know what it is.
+;BUFFER_LEN = 10000
 ;;
-;; The router will log different things at different levels.
+;; Sub logger modes, a single comma means use default MODE above, empty means disable it
+;logger.access.MODE=
+;logger.router.MODE=,
+;logger.xorm.MODE=,
```

Comment:
Maybe we can rename it as `logger.sql.MODE`?

Reply:
The logger itself was hard-coded to "xorm" in code.

If you agree to rename all GetLogger("xorm") in code, we can change it.

Do you prefer to keep it as is, or rename all?

Reply:
Since this PR removed the key `XORM`, I think we can rename it to another one.

Reply:
Another question , is "sql" a proper name for this logger?

This logger is only used by "xorm", and I guess not only SQLs, it also include other XORM messages.

Personally I think the name "xorm" is not bad.